src/main/java/org/spongepowered/api/scoreboard/Score.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * This file is part of Sponge, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered.org <http://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.api.scoreboard;
+
+import com.google.common.base.Optional;
+import org.spongepowered.api.scoreboard.objective.Objective;
+import org.spongepowered.api.text.message.Message;
+
+/**
+ * A score entry for an {@link Objective}. Changing this will not affect any other
+ * objective or scoreboard.
+ */
+public interface Score {
+
+    /**
+     * Gets the name of the entry being tracked by this score.
+     *
+     * @return The name of this score's entry
+     */
+    Message getEntry();
+
+    /**
+     * Gets the {@link Objective} this score belongs to, if it hasn't been removed.
+     *
+     * @return The {@link Objective} that owns this score, if available
+     */
+    Optional<Objective> getObjective();
+
+    /**
+     * Gets the current score value.
+     *
+     * @return The current score value
+     */
+    int getScore();
+
+    /**
+     * Sets the current score value.
+     *
+     * @param score The new score value
+     */
+    void setScore(int score);
+
+    /**
+     * Removes this score from its {@link Objective}.
+     */
+    void remove();
```

Comment:
Isn't removing a `Score` actually an operation on the `Objective` and not the `Score`? I think `Objective.removeScore(score)` would be more logical